build/ol/install-packages.sh

```diff
@@ -53,6 +53,9 @@ curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.s
 dnf update -y
 dnf install -y git-lfs
 
+# Install missing language pack for en_US.UTF-8 (see issue #2487)
+dnf install -y glibc-langpack-en
```

Comment:
Can you add this to the install that starts on line 27?

Reply:
Done in d8fe21f489b2b9a3b2b07ed8257f9dfac9cc224f.